### ISSUE TITLE
border.css: CSS arrow border

### ISSUE THREAD

**Issue by strarsis** [atomiks/tippyjs#902]
## Bug description
When using the helper style `border.css` the CSS arrows don't get a second pseudoelement for the arrow border.
This means that while the tooltip has a border, but the arrow has no border.
This is solved differently with the SVG arrow, where two SVG arrows are used in the example.

**Comment by atomiks** [comment]
The regular CSS arrows should have a border with that sheet. You can see in the visual test for it

**Comment by strarsis** [comment]
@atomiks: One thing confuses me: Why is the CSS not explictly added as external resource in the [Codepen issue template](https://codepen.io/atomiks/pen/yvwQyZ)?
Does the bundled version (not ES imported one) automatically inject the styles?

**Comment by atomiks** [comment]
Okay. We should amend the docs to show an example in this case

**Comment by williams-aguilar** [comment]
Anyone has a working example on how to increase size using pixels.

**Comment by silkfire** [comment]
I agree. It's very frustrating that the documentation vaguely explains what needs to be done but doesn't actually show you an example of how to achieve it. Still no update and it's almost the end of 2024.
